src/radiation/radiation.hpp

```diff
@@ -0,0 +1,105 @@
+//
+// Created by owen on 3/19/22.
+//
+#ifndef ABLATELIBRARY_RADIATION_HPP
+#define ABLATELIBRARY_RADIATION_HPP
+
+#include <memory>
+#include <set>
+#include "solver/cellSolver.hpp"
+#include "solver/timeStepper.hpp"
+
+namespace ablate::radiation {
+
+class RadiationSolver : public solver::CellSolver, public solver::RHSFunction {  //Cell solver provides cell based functionality, right hand side function compatibility with finite element/ volume
+   public:
+    PetscInt raynumber;
+
+    /**
+     * Boundary information.
+     */
+    typedef struct {
```

Comment:
Are you using this struct?

Reply:
I thought that this is where the cell center location is accessed from. If that isn't the case then I wouldn't need it.